test/e2e/config/azure-dev.yaml

```diff
@@ -145,6 +151,8 @@ variables:
   AKS_KUBERNETES_VERSION: "latest"
   KUBERNETES_VERSION: "${KUBERNETES_VERSION:-stable-1.23}"
   ETCD_VERSION_UPGRADE_TO: "3.5.3-0"
+  PRE_1_23_KUBERNETES_VERSION: "v1.22.4"
```

Comment:
what is this / where is it used?

Reply:
It was used earlier. Let me remove that.

Reply:
Fixed it.